src/main/java/monique/parser/Parser.java

```diff
@@ -20,136 +20,135 @@
-                        if (!hasSecondWord) {
-                            throw new ParseException();
-                        }
-                        int taskNum = Integer.parseInt(fullCommand.split("mark ")[1]) -1;
-                        command = new MarkCommand(taskNum);
-                        break;
-                    } catch (ParseException pe) {
-                        pe.advice();
-                    } catch (NumberFormatException nfe) {
-                        System.out.println("you have tried to use an invalid number");
-                    } finally {
-                        break;
-                    }
-                }
-                case "unmark": {
-                    try {
-                        if (!hasSecondWord) {
-                            throw new ParseException();
-                        }
-                        int taskNum = Integer.parseInt(fullCommand.split("unmark ")[1]) -1;
-                        command = new UnmarkCommand(taskNum);
-                        break;
-                    } catch (ParseException pe) {
-                        pe.advice();
-                    } catch (NumberFormatException nfe) {
-                        System.out.println("you have tried to use an invalid number");
+            }
+            case "unmark": {
+                try {
+                    if (!hasSecondWord) {
+                        throw new ParseException();
                     }
-                }
-                case "/commands": {
-                    command = new GuideCommand();
+                    int taskNum = Integer.parseInt(fullCommand.split("unmark ")[1]) - 1;
+                    command = new UnmarkCommand(taskNum);
                     break;
+                } catch (ParseException pe) {
+                    pe.advice();
+                } catch (NumberFormatException nfe) {
+                    System.out.println("you have tried to use an invalid number");
                 }
-                case "delete" : {
-                    try {
-                        if (!hasSecondWord) {
-                            throw new ParseException();
-                        }
-                        int taskNum = Integer.parseInt(fullCommand.split("delete ")[1]) -1;
-                        command = new DeleteCommand(taskNum);
-                        break;
-                    } catch (ParseException pe){
-                        pe.advice();
-                    } catch (NumberFormatException nfe) {
-                        System.out.println("you have tried to use an invalid number");
+            }
+            case "/commands": {
+                command = new GuideCommand();
+                break;
+            }
+            case "delete": {
+                try {
+                    if (!hasSecondWord) {
+                        throw new ParseException();
                     }
+                    int taskNum = Integer.parseInt(fullCommand.split("delete ")[1]) - 1;
```

Comment:
same here, the -1 due to line and array difference can be in the form of a constant to improve readability